student.rb

```diff
@@ -0,0 +1,8 @@
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @classroom = nil
+  end
+end
```

Comment:
- Kindly, Check for a method that serves as the 'belongs-to' side (a student belongs to a classroom). check [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for more info :+1:
```
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```